css/main.css

```diff
@@ -22,7 +22,7 @@ header {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 15px;
+  /* margin-left: 15px; */
```

Comment:
- [ ] _[Required]_ Please remove commented-out styles from your CSS as it makes the file clustered and hard to read.